cmd/podman/ps.go

```diff
@@ -190,6 +191,7 @@ func psInit(command *cliconfig.PsValues) {
 	flags.BoolVarP(&command.Size, "size", "s", false, "Display the total file sizes")
 	flags.StringVar(&command.Sort, "sort", "created", "Sort output by command, created, id, image, names, runningfor, size, or status")
 	flags.BoolVar(&command.Sync, "sync", false, "Sync container state with OCI runtime")
+	flags.UintVarP(&command.Watch, "watch", "w", 1, "Watch the ps output on a interval in seconds")
```

Comment:
Why default to 1? I think we want to default to 0 here, for not following

Reply:
I agree.